proseco/acq.py

```diff
@@ -306,20 +306,19 @@ def get_acq_candidates(self, stars, max_candidates=20):
         self.log(f'Reduced star list from {len(stars)} to '
                  f'{len(cand_acqs)} candidate acq stars')
 
-        # Reject any candidate with a spoiler that is within a 30" HW box
-        # and is within 3 mag of the candidate in brightness.  Collect a
-        # list of good (not rejected) candidates and stop when there are
-        # max_candidates.
+        # Reject any candidate with a spoiler or bad star.  Collect a list of
+        # good (not rejected) candidates and stop when there are
+        # max_candidates.  Check for col spoilers only against stars that are
+        # bright enough and on CCD
```

Comment:
These are still col spoilers that can be up to 200 arsec off the CCD right?  That seems fine.

Reply:
As long as the candidate star and possible spoiler have column values being compared, then having slightly more stars here is formally correct.  Just slower by a miniscule amount.  So the comment should be improved.